frontend/packages/kubevirt-plugin/src/components/modals/delete-device-modal/delete-device-modal.tsx

```diff
@@ -12,10 +12,8 @@ import {
 import { k8sPatch } from '@console/internal/module/k8s';
 
 import { getName } from '@console/shared';
-import { TemplateModel } from '@console/internal/models';
-import { VirtualMachineModel } from '../../../models';
 import { VMLikeEntityKind } from '../../../types';
-import { isVm } from '../../../selectors/selectors';
+import { getVmLikeModel } from '../../../selectors/selectors';
```

Comment:
Side note: we should change this to `getVMLikeModel` (capitalized acronyms) if possible.